File: 0009-palindrome-number/0009-palindrome-number.py
class Solution(object):
    def isPalindrome(self, x):
        """
        :type x: int
        :rtype: bool
        """
        t=abs(x)
        rev=0
        while(x>0):
            rem=x%10
            rev=rev*10+rem
            x=x//10
        if(t==rev):
            return True
        else:
            return False


        

File: 0009-palindrome-number/0009-palindrome-number.cpp
class Solution {
public:
    bool isPalindrome(int x) {
        // int tmp=x;
        // int rev=0;
        // if(tmp<0) return false;
        // while(tmp>0){
        //     int ld=tmp%10;
        //     rev=(rev*10)+ld;
        //     tmp/=10;
        // }
        // if(x==rev) return true;
        // return false;
        string tmp=to_string(x);
        string tmp2=to_string(x);
        reverse(tmp2.begin(),tmp2.end());
        return tmp==tmp2;
    }
};